src/auth/socialauth.js

```diff
@@ -0,0 +1,35 @@
+import Expo from 'expo';
+import { auth, facebookAuthProvider, googleAuthProvider } from '../firebase/initialize';
+
+export async function signInWithGoogleAsync() {
+  const result = await Expo.Google.logInAsync({
+    androidClientId: '506898842953-a5djvc12er7cbmv78ajfjidokjmlropn.apps.googleusercontent.com',
```

Comment:
I imagine we don't want to leave these hardcoded? You might want to pull the keys out into a constants/keys file so that we remember to switch them to env variables

Reply:
Yeah, that's a good call